performance/src/main/java/org/hibernate/validator/performance/cascaded/CascadedValidation.java

```diff
@@ -50,19 +67,22 @@ public void testCascadedValidation() {
 		piggy.addFriend( kermit ).addFriend( gonzo );
 		gonzo.addFriend( kermit ).addFriend( piggy );
 
-		Set<ConstraintViolation<Person>> violations = validator.validate( kermit );
+		Set<ConstraintViolation<Person>> violations = state.validator.validate( kermit );
 		assertThat( violations ).hasSize( 0 );
 	}
 
-	/**
-	 * To be executed manually. Not part of the JMeter tests for now.
-	 */
-	@Test
-	public void testCascadedValidationMultiThreaded() throws Exception {
+//	@Benchmark
+//	@BenchmarkMode(Mode.All)
+//	@OutputTimeUnit(TimeUnit.MILLISECONDS)
+//	@Fork(value = 1)
+//	@Threads(2)
+//	@Warmup(iterations = 10)
+//	@Measurement(iterations = 50)
+	public void testCascadedValidationMultiThreaded(CascadedValidationState state) throws Exception {
```

Comment:
Is this one a good benchmark test ? as you can see previously it was marked as one to be executed manually. And based on what it's doing what would be a difference between this and just running a benchmark on testCascadedValidation (which is called in the end of current test anyway) ? 
I've commented this one out as it is taking to much time to execute and with previous configuration of 100/50 treads it even hasn't been able to run as it was creating too much treads and VM just wouldn't allow it.